src/main/java/org/broadinstitute/dsm/model/birch/RegisterBirchOrderCLI.java

```diff
@@ -0,0 +1,138 @@
+package org.broadinstitute.dsm.model.birch;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.util.Map;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import org.apache.commons.dbcp2.PoolableConnection;
+import org.apache.commons.dbcp2.PoolingDataSource;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.dsm.careevolve.Authentication;
+import org.broadinstitute.dsm.careevolve.Covid19OrderRegistrar;
+import org.broadinstitute.dsm.careevolve.OrderResponse;
+import org.broadinstitute.dsm.careevolve.Patient;
+import org.broadinstitute.dsm.careevolve.Provider;
+import org.broadinstitute.dsm.cf.CFUtil;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.DdpKit;
+import org.broadinstitute.dsm.statics.ApplicationConfigConstants;
+import org.broadinstitute.dsm.statics.DBConstants;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+import org.elasticsearch.client.RestHighLevelClient;
+
+/**
+ * CLI utility for manually placing orders in CareEvolve (and birch)
+ * in response to BQMS tickets for kits that have been returned
+ * without UPS shipping info.
+ */
+public class RegisterBirchOrderCLI {
+
+    private static final String USAGE = "args in order are hruid, kitLabel, externalOrderNumber, and collectionTime\n" +
+            "for example java -Dconfig.file=... RegisterBirchOrderCLI P1234 TBOS-123 12345678910111213 06/30/2020 15:34\n" +
+            "Make sure you are on a broad network\n";
+    public static final String DATE_FORMAT = "MM/dd/yyyy hh:mm";
+
+
+    public static void main(String[] args) {
+        try {
+            if (args == null || args.length == 0 || ("-h".equals(args[0]))) {
+                System.out.println(USAGE);
+                System.exit(0);
+            }
+            String participantHruid = args[0];  // user.hruid
+            String kitLabel = args[1]; // ddp_kit.kit_label
+            String externalOrderNumber = args[2];  // ddp_kit_request.external_order_number
+            /** {@link DATE_FORMAT} */
+            String collectionTime = args[3];
+
+            Instant collectionDate = null;
+            try {
+                collectionDate = new SimpleDateFormat(DATE_FORMAT).parse(collectionTime).toInstant();
```

Comment:
Do we need to change that to Eastern time zone?

Reply:
That is a great idea!  It's defaulting to that, but only because of my laptop's timezone...

Reply:
Is this the way to do it?
```java
LocalDateTime.parse(collectionTime, DateTimeFormatter.ofPattern(DATE_FORMAT))
    .atZone(ZoneId.of("America/New_York"))
    .toInstant();
````